lib/multiplex-mappers.cc

```diff
@@ -153,14 +153,16 @@ class SpiralMultiplexMapper : public MultiplexMapperBase {
 
 class ZStripeMultiplexMapper : public MultiplexMapperBase {
 public:
-  ZStripeMultiplexMapper(const char *name, int even_vblock_offset, int odd_vblock_offset)
+  ZStripeMultiplexMapper(const char *name, int even_vblock_offset, int odd_vblock_offset, int width, int height)
   : MultiplexMapperBase(name, 2),
     even_vblock_offset_(even_vblock_offset),
-    odd_vblock_offset_(odd_vblock_offset) {}
+    odd_vblock_offset_(odd_vblock_offset),
+    width_(width),
+    height_(height) {}
 
   void MapSinglePanel(int x, int y, int *matrix_x, int *matrix_y) const {
-    static const int tile_width = 8;
-    static const int tile_height = 4;
+    static const int tile_width = width_;
```

Comment:
if you use width_ and height_ member variables, you can't assign this to static variables here, as they are global and override them with whoever ran last.

Just replace all uses of `tile_height`, and `tile_width` with the new members `height_` and `width_`.